lib/fog/compute/xen_server/real.rb

```diff
@@ -44,7 +44,31 @@ def connection_host
         def event
           @event ||= Fog::Compute::XenServer::Models::Event.new(service: self)
         end
+
+        private
+
+        def load_data(keep_type = nil, delete_templates = false)
```

Comment:
Fair call; I'll trust you.
To be _really_ honest, I only have a simple understanding of what this whole gem does — saw some warnings and thought I could apply some standard code practices 😆
I'll do a little more (hopefully over the next day) and will review this again.